src/ui/public/doc_table/doc_table.html

```diff
@@ -20,6 +20,12 @@
           class="discover-table-row"></tr>
       </tbody>
     </table>
+    <div ng-if="hits.length === sampleSize && page.last" class="discover-table-footer">
```

Comment:
Oh the other thing is that in the discover footer now, there is a `back to top` link where the logic is in discover, not in doc table.  I got rid of that, since it doesn't make sense esp if this error message goes on top, not on the bottom.